### PR TITLE
update some cfgs to get rid of the annoying warnings

### PR DESCRIPTION
**Pull request by hsqStephenZhang**
i donnot know if this is the best way of removing the warnings, maybe #[cfg(allow ...)] is a better idea? I'd love to do this job for this project.

### CODE REVIEW

**Comment by alessandrod** [comment]
Hey, thanks for the PR! What warnings do you get running what command?

File: aya/src/sys/mod.rs [hunk 1]
```diff
@@ -5,11 +5,14 @@ mod perf_event;
 #[cfg(test)]
 mod fake;
 
+#[cfg(not(test))]
+use libc::utsname;
+use libc::{c_int, c_long, pid_t};
 #[cfg(not(test))]
```
Comment (alessandrod) [inline aya/src/sys/mod.rs:11]: please remove this

File: aya/src/sys/mod.rs [hunk 2]
```diff
@@ -5,11 +5,14 @@ mod perf_event;
...
 use std::convert::TryInto;
-use std::{ffi::CString, io, mem};
-
-use libc::{c_int, c_long, pid_t, utsname};
+use std::io;
+#[cfg(not(test))]
+use std::{ffi::CString, mem};
```
Comment (alessandrod) [inline aya/src/sys/mod.rs:15]: and add `convert::TryInto` here

Comment (hsqStephenZhang) [reply aya/src/sys/mod.rs]: i got it, it's done now

File: aya/src/sys/mod.rs [hunk 3]
```diff
@@ -6,19 +6,17 @@ mod perf_event;
 mod fake;
 
 #[cfg(not(test))]
```
Comment (alessandrod) [inline aya/src/sys/mod.rs:8]: please remove this to fix the tests

Comment (hsqStephenZhang) [reply aya/src/sys/mod.rs:9]: sorry, it's my fault, the tests is clean now 😂

**Comment by alessandrod** [comment]
Thank you! 🙏

**Pull request merged**